tests/publiccloud/azure_aitl.pm

```diff
@@ -0,0 +1,129 @@
+    my $aitl_image_name = $provider->generate_azure_image_definition();
+
+    my $aitl_get_options = "-s $subscription_id -r $resource_group -n $aitl_job_name";
+
+    my $openqa_url = get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));
+    my $created_by = "$openqa_url/t$job_id";
+    my $tags = "openqa-aitl=$job_id openqa_created_by=$created_by openqa_var_server=$openqa_url";
+
+    # Get the AITL script
+    assert_script_run("curl https://raw.githubusercontent.com/microsoft/lisa/refs/tags/$aitl_client_version/microsoft/utils/aitl/aitl.py -o /tmp/aitl.py");
+
+    # Configure default location and create Resource group
+    assert_script_run("az configure --defaults location=$region");
+    assert_script_run("az group create -n $resource_group --tags '$tags'");
+
+    # Get manifest from data folder
+    assert_script_run("curl " . data_url("publiccloud/aitl/$aitl_manifest") . " -o /tmp/$aitl_manifest");
+    assert_script_run("sed -i -e 's/<IMAGE_NAME>/$aitl_image_name/g' -e 's/<IMAGE_VERSION>/$aitl_image_version/g' -e 's/<IMAGE_GALLERY_NAME>/$aitl_image_gallery/g' /tmp/$aitl_manifest");
+
+    # Create AITL Job based on a manifest
+    assert_script_run("python3.11 /tmp/aitl.py job create $aitl_get_options -b @/tmp/$aitl_manifest");
+
+    # Wait a few seconds to give Azure time to create the jobs
+    sleep(10);
+
+    my $status = script_output(qq(python3.11 /tmp/aitl.py job get $aitl_get_options -q "properties.results[].status|{RUNNING:length([?@=='RUNNING']),QUEUED:length([?@=='QUEUED'])}"));
+
+    # Remove the first two non-JSON lines from the status JSON.
+    $status =~ s/^(?:.*\n){1,3}//;
+    my $status_data = decode_json($status);
+
+    # Get AITL job status
+    # AITL Jobs run in parallel so it's possible to have Jobs in all kind of states.
+    # The goal of the loop is to check there are no Jobs Queued or currently Running.
+    while ($status_data->{RUNNING} > 0 || $status_data->{QUEUED} > 0) {
+        sleep(30);
+        $status = script_output(qq(python3.11 /tmp/aitl.py job get $aitl_get_options -q "properties.results[].status|{RUNNING:length([?@=='RUNNING']),QUEUED:length([?@=='QUEUED'])}"));
+        $status =~ s/^(?:.*\n){1,3}//;
+        $status_data = decode_json($status);
+        print("Unfinished AITL Jobs! Running:", $status_data->{RUNNING}, " QUEUED: ", $status_data->{QUEUED});
+    }
+
+    # Need to save results to a variable
+    my $results = script_output("python3.11 /tmp/aitl.py job get $aitl_get_options -q 'properties.results[]'");
+
+    # Remove the first two non-JSON lines from the results JSON.
+    $results =~ s/^(?:.*\n){1,3}//;
+
+    # Convert to JUnit XML and upload to host
+    my $extra_log = json_to_xml($results, $aitl_image_name);
+
+    # Download file from host pool to the instance
+    assert_script_run('curl -s ' . autoinst_url('/files/aitl_results.xml') . ' -o /tmp/aitl_results.xml');
+    parse_extra_log('XUnit', '/tmp/aitl_results.xml');
+
+}
+
+sub json_to_xml {
```

Comment:
Hi @mdoucha . AFAIK you was involved in writing parser for LTP tests . Can you please review this part from POV "Extra results tab parsers" ? 

CC: @foursixnine , @perlpunk